package.json

```diff
@@ -87,7 +87,7 @@
     "webpack-merge": "^5.8.0"
   },
   "dependencies": {
-    "@centreon/centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend#develop",
+    "@centreon/centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
```

Comment:
Are you going to wait until it is merged into centreon-frontend or will you have to think about doing it later?